private/Producers/FbxProducer/FbxProducerImpl.cpp

```diff
@@ -954,64 +958,126 @@ cd::BoneID FbxProducerImpl::AddBone(const fbxsdk::FbxNode* pSDKNode, cd::BoneID
 
 void FbxProducerImpl::ProcessAnimation(fbxsdk::FbxScene* scene, cd::SceneDatabase* pSceneDatabase)
 {	
+	// Locates all skeleton nodes in the fbx scene. Some might be nullptr.
+	std::vector<FbxNode*> bones;
+	for (int i = 0; i < pSceneDatabase->GetBoneCount(); i++) 
```

Comment:
```suggestion
	for (uint32_t boneIndex = 0U; boneIndex < pSceneDatabase->GetBoneCount(); ++boneIndex) 
```
Reasons:
1. The name "boneIndex" is meaningful than "i". Actually, "i" is a bad pratice which populates by some C++ books.
2. uint32_t is better because bone count is never less than 0.